### PR TITLE
Update html language attribute

### PR DESCRIPTION
**Pull request by daun**
What is says on the tin: update the document language on navigation.

`<html lang="en-US">` → `<html lang="fr-FR">`

Let me know if you can think of a cheaper way of getting the new `lang` attribute. Parsing the new page as a DOM fragment might be slow (?), but I mistrust regular expressions too much to take that shortcut.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -103,4 +105,19 @@ export default class HeadPlugin extends Plugin {
 
 		return addTags;
 	};
+
+	updateHtmlLangAttribute = () => {
+		const html = document.documentElement;
+
+		const newPage = new DOMParser().parseFromString(
```
Comment (gmrchk) [inline src/index.js:112]: Didn't know `DOMParser` existed, nice!

**Comment by gmrchk** [comment]
I think the implementation is great. The DOM parsing is a bit unfortunate, but I don't think there will be any real problems with performance, and we should probably go with the safer option, which it is. 

My main concern is, is the Head plugin the right place for this functionality? It doesn't even have to do anything with `head` tag. Could it be a separate plugin instead maybe?

**Comment by daun** [comment]
There is [a plugin](https://www.npmjs.com/package/@mashvp/swup-html-lang-plugin) that solves this exact problem. If you feel it's best to not include this feature here, fair enough.

However, I'd argue that this should be part of the head plugin — not because it's _technically_ part of the head tag, but because it's in line with its _purpose_ of updating meta tags on page visits. It just so happens the lang attribute is not a literal `meta` tag, but that's a technicality.

**Comment by gmrchk** [comment]
Sorry for the delay. Consider me convinced. [Merged and published](https://github.com/swup/head-plugin/commits/master) manually.

**Pull request closed without merging**